### DISCUSSION TITLE
Simple Feature Ideas

### DISCUSSION THREAD

**Discussion by PairOfTrousers** [chesspecker/chesspecker#3] [General]
Really enjoying chesspecker, thanks for developing it. I'd like to suggest some ideas that would make it more helpful that shouldn't add too much code or data storage.

1. How many times have I completed a set of puzzles?
2. Instead of just showing my best time, what are my first, best, and most recent times?
3. Currently the accuracy appears to display overall accuracy. It would be great if it would reflect the most recent attempt or even the best and most recent attempts.
4. Display how many puzzles are in each set.
5. Could you have a display for mobile that shows progress within the set? Oftentimes I am trying to complete a set in a sitting but on mobile I have no idea how much is left.
6. Could there be an option to edit the names of the sets?

Thanks again.

**Comment by zeezop** [comment]
Agree with the suggestions. As an example for #3, When I exited without completing all the exercises it shows an accuracy of 59%. Intuitively I would expect to see 100% as so far I have zero errors.

**Comment by pnodet** [comment]
**Hi guys! We are currently implementing a dashboard page to view the statistics on each of your set. You can checkout this issue : #6 to learn more of what we are building. Feel free to comment here any new suggestions you may have.**

> How many times have I completed a set of puzzles? _From @PairOfTrousers_ 

Obviously, a must have.

> Display how many puzzles are in each set.

Yes.

> Could there be an option to edit the names of the sets?

Yes.

> Instead of just showing my best time, what are my first, best, and most recent times?

You will be able to compare your last run agains your first and you average to see you progress

> Could you have a display for mobile that shows progress within the set? Oftentimes I am trying to complete a set in a sitting but on mobile I have no idea how much is left.

Currently working on it! Will be included

> Agree with the suggestions. As an example for https://github.com/chesspecker/chesspecker/discussions/3, When I exited without completing all the exercises it shows an accuracy of 59%. Intuitively I would expect to see 100% as so far I have zero errors. _From @zeezop_ 
> Currently the accuracy appears to display overall accuracy. It would be great if it would reflect the most recent attempt or even the best and most recent attempts.

**The accuracy measurement will not be part of the next release. We are replacing it by a grade for each puzzle you complete, ranking from A to F (A showing perfect completion and F a completed puzzles with many mistakes)**

**Reply by cshancock** [reply]
Can the letter grades at least show what accuracies are associated with each grade? I make myself complete a set at >= 85% accuracy before moving on to a more difficult set on the theme. The 85% figure comes from an academic paper (that I am having trouble finding!) and is also the figure recommended by Dan Heisman in his books.

**Reply by pnodet** [reply]
Sure
